lib/fathead/scala/parse.py

```diff
@@ -0,0 +1,52 @@
+from bs4 import BeautifulSoup
+import os
+
+BASE_DIRECTORY = "./download/api/scala"
+BASE_URL = "https://www.scala-lang.org/api/current/"
+
+
+def get_all_files():
+	doc_files = []
+
+	for (path, dirs, files) in os.walk(BASE_DIRECTORY):
+		for f in files:
+			doc_files.append( "%s/%s" % (path, f))
+	return doc_files
+
+def get_docs(filename):
+	if filename[-5:] == ".html":
+		soup = BeautifulSoup(open(filename), 'html.parser')
+		title_link = soup.find_all("h1")[0]
+		title_text = title_link.text
+		
+		url = BASE_URL + filename.replace("./download/api/", "") #title_link.find("a").get('href')
+
+		article_text = soup.find_all("div", attrs={"class" : "comment"})[0].text
```

Comment:
@sam09 we should pull out code blocks from the docs and put them into `<pre><code>` so we can display them properly. Likewise and text content needs to be put inside a `<p>`. That way we can closely mimic the display in the docs 👍 

Otherwise we're showing code as plaintext without formatting which looks ugly and is hard to read